alica_engine/src/engine/TeamObserver.cpp

```diff
@@ -111,7 +111,7 @@ void TeamObserver::tick(RunningPlan* root)
         }
 
         if (root->recursiveUpdateAssignment(updatespts, activeAgents, noUpdates, time)) {
```

Comment:
Do we need a side effects of this call? If yes, I'd prefer to move it outside of if condition, if no let's eliminate it